web/themes/new_weather_theme/assets/sass/components/location-search.scss

```diff
@@ -1,11 +1,18 @@
 /* stylelint-disable scss/at-extend-no-missing-placeholder */
+@use "uswds-core" as *;
 @forward "uswds-theme";
 @forward "uswds";
 
 .wx-combo-box {
   @extend .usa-combo-box;
 }
 
+.wx-combo-box li[role="option"] {
+  &:hover {
```

Comment:
```suggestion
  &:not(.wx-combo-box__list-option--focused):hover {
```

This change preserves the styling of the focused element when it is hovered. Otherwise, this makes the focused element unreadable when it's hovered:

![image](https://github.com/weather-gov/weather.gov/assets/142943695/a8d3f5b5-b397-425b-8b39-62287dc89003)

Though perhaps a better solution would be to target the text color if the hovered item is focused. 🤷‍♂️ In any case, this is a super minor, easy change, so whatever makes the best sense.